### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by JonasHelming**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<div id="theia-logo" align="left">
+    <br />
+    <img src="https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/EF_GRY-OR_svg.svg?sanitize=true" alt="Eclipse Logo" width="300"/>
+</div>
+
+# Community Code of Conduct
+
+Version 1.1
+
+October 21, 2019
```
Comment (marcdumais-work) [inline CODE_OF_CONDUCT.md]: Use newer version?

**Comment by JonasHelming** [comment]
Yes, good catch. Is it maybe better to only have the content in the main repo and add links to the files in all other repos? WDYT?

**Comment by marcdumais-work** [comment]
> Yes, good catch. Is it maybe better to only have the content in the main repo and add links to the files in all other repos? WDYT?

That would be simpler to manage/update for sure. I think the foundation expects certain files to be present in every project repository, but perhaps this could be stretched a little. Maybe that's what you had in mind, but I would agree having the files, like "CODE_OF_CONDUCT.md" present in every repo, but containing only a link to the same file in the main Theia repo?

**Review by marcdumais-work — APPROVED** [review]
LGTM! 

Note: it would probably be good to update the CoC in the main repo eventually, but that's outside the scope of this review here :)

**Pull request merged**